lib/stations/core/A6s_Railway_ParallelExecution_StationHandler.ts

```diff
@@ -37,24 +38,26 @@ export class A6s_Railway_ParallelExecution_StationHandler extends BaseStationHan
         resolvers: A6sRailwayResolverRegistry,
         parentsPath: string[]
     ): Promise<void> {
-        const errors: Error[] = [];
+        const parallelException = new ParallelException();
         const promises = options.map(async (s: IRailWayStation): Promise<void> => {
             try {
                 await this.a6sRailwayUtil.processStation(s, handlers, resolvers, parentsPath);
             } catch (e) {
-                errors.push(new Error([
+                e.message = [
```

Comment:
This message is nowhere longer used. Just pass error directly to parallelException.addException